tests/upgrades/test_satellitesync.py

```diff
@@ -14,138 +14,120 @@
 
 :Upstream: No
 """
-from fauxfactory import gen_string
 from nailgun import entities
 from upgrade_tests import post_upgrade
 from upgrade_tests import pre_upgrade
-from upgrade_tests.helpers.scenarios import create_dict
-from upgrade_tests.helpers.scenarios import get_entity_data
 
 from robottelo import ssh
 from robottelo.cli.contentview import ContentView
 from robottelo.cli.package import Package
-from robottelo.test import CLITestCase
 
 
-class scenario_preversion_cv_exports_imports(CLITestCase):
```

Comment:
I think we need to modify the design here no need to keep the class just for reading the scenarios. As we know the post-upgrade test case depends on pre-upgrade and it executes only after the pre-upgrade gets passed, so I think we can also skip uploading the pre-upgrade test results. We should need to target the post-upgrade test results only and each test case has proper docstring and I think that would be sufficient to understand the whole scenarios. 
What do you say?

Reply:
@devendra104 We should discuss this over a call. Lets meet.

Reply:
@devendra104 As per discussion on the chat/Meet. I defiantly agree that we can rely on post-upgrade part completely to validate the scenario is pass/fail and of-course as you said above, the docstrings in postupgrade only can help us to upload the scenario as a Test and its results in Polarion.

But my only intention here is there should be something that holds the pre and post funcs as a `Upgrade Scenario` and that is achievable with putting pre and post test_functions in a one-liner class definition. This is not going to degrade or upgrade the performance but this will help to identify a one scenario easily. The code will also look more formatted and ease on eyes. 

Hope that helps you.

Reply:
Fixed.